### PR TITLE
Some checks to prevent kicking/banning/quieting the bot

### LINKED ISSUE #6: kick/ban/quiet can be used against the bot
A safeguard would be good.

### PR DESCRIPTION
**Pull request by xPaw**
Fixes #6 and disallows hostmasks in kick command to prevent an error.

### CODE REVIEW

File: abbott/plugins/admin.py [hunk 1]
```diff
@@ -507,6 +507,17 @@ def kick(self, event, match):
...
 
+        # TODO: Do a whois?
+        if "@" in nick or "!" in nick or "$" in nick:
+            event.reply("Can't kick by a hostmask. Sorry.")
+            return
+
+        mynick = (yield self.transport.issue_request("irc.getnick"))
+
+        if mynick.lower() == nick.lower():
+            nick = event.user.split("!",1)[0];
```
Comment (zuzak) [inline abbott/plugins/admin.py]: Missing whitespace after ',' (pep8)

Comment (zuzak) [inline abbott/plugins/admin.py]: And it ends in a semi-colon; what sort of barbaric language do you think this is

File: abbott/plugins/admin.py [hunk 2]
```diff
@@ -795,6 +818,12 @@ def ban(self, event, match):
             event.reply("Malformed hostmask. Hostmask syntax is <nick>!<username>@<host>", direct=True, notice=True)
             return
 
+        targetnick = hostmask.split("!",1)[0];
```
Comment (zuzak) [inline abbott/plugins/admin.py]: missing whitespace after ',', semi-colon of doom

Comment (xPaw) [inline abbott/plugins/admin.py]: Original code doesn't really follow pep8, I just copied how it was done before. If you want to be pedantic, make a PR that fixes all files to follow pep8.

**Pull request merged**